### PR TITLE
Update version banner

### PR DESCRIPTION
**Pull request by heitortsergent**
## What?

Update the version banner for the JavaScript API section to display on latest version, and update link to point to Grafana docs.

## Checklist

Please fill in this template:
- [x] I have used a meaningful title for the PR.
- [x] I have described the changes I've made in the "What?" section above.
- [x] I have performed a self-review of my changes.
- [x] I have run the `npm start` command locally and verified that the changes look good.

### CODE REVIEW

**Comment by heitortsergent** [comment]
@ppcano I think I was able to figure out the version banner, let me know how it looks! 🙏

File: src/components/shared/page-info/page-info.module.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .container {
   margin-bottom: 20px;
-  margin-top: 115px;
+  margin-top: 165px;
```
Comment (ppcano) [inline src/components/shared/page-info/page-info.module.scss:3]: Screenshot 2023-11-29 at 23 51 02


On my screen, the margin seems too large.

**Review by ppcano — COMMENTED** [review]
Commented about the `margin-top` changes. The rest looks perfect!

File: src/components/shared/page-info/page-info.module.scss (hunk 1 above)
Comment (heitortsergent) [reply src/components/shared/page-info/page-info.module.scss]: Ah good catch, thanks @ppcano! I just updated the PR, I think it's fixed now so the higher `margin-top` only applies to the JS API section.

**Pull request merged**